extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -105,6 +111,16 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
       StateProperty.PANNING_MEDIA_STATE,
       (panningMediaState) => this.onPanningMediaStateChange_(panningMediaState)
     );
+    // Mutation observer for distance attribute
```

Comment:
Why is this not triggered with the `CURRENT_PAGE_ID` state?

Reply:
It's asynced so it doesn't block page transition.

Reply:
What does this callback do that's CPU heavy? To me it seems to just set the `this.distance_` property, and doesn't call any time-taking function

Reply:
If it's called with the `CURRENT_PAGE_ID` state it will get an old value since `CURRENT_PAGE_ID` is set before the distance attribute is updated.